File: MA/FuiaMihaiJetpackCompose-master/app/src/main/java/domain/dish.kt
package domain

import android.media.Image

enum class result_type{
    CATASTRPHE,FAILURE,OK,SUCCESS,MAGNUM_OPUS
}
class dish {
    var ID: Long? = 0
        get() = field
        set(value){
            field=value
        }
    var Name: String = ""
        get() = field
        set(value){
            field=value
        }
    var Ingredients: String = ""
        get() = field
        set(value){
            field=value
        }
    var Recipe: String = ""
        get() = field
        set(value){
            field=value
        }
    var Results: result_type = result_type.OK
        get() = field
        set(value){
            field=value
        }
    var Image: Int = 0
        get() = field
        set(value){
            field=value
        }

    constructor(
        ID: Long?,
        Name: String,
        Ingredients: String,
        Recipe: String,
        Results: result_type,
        Image: Int
    ) {
        this.ID = ID
        this.Name = Name
        this.Ingredients = Ingredients
        this.Recipe = Recipe
        this.Results = Results
        this.Image = Image
    }

    constructor(
    ) {
        this.ID = -1
    }


}

File: MA/flutter_ma/lib/domain/dish.dart


import 'package:flutter/cupertino.dart';

enum  result_type{
CATASTROPHE,FAILURE,OK,SUCCESS,MAGNUM_OPUS
}

class dish{
  int _ID;

  int get ID => _ID;

  set ID(int ID) {
    _ID = ID;
  }
  String _Name;

  String get Name => _Name;

  set Name(String Name) {
    _Name = Name;
  }
  String _Ingredients;

  String get Ingredients => _Ingredients;

  set Ingredients(String Ingredients) {
    _Ingredients = Ingredients;
  }
  String _Recipe;

  String get Recipe => _Recipe;

  set Recipe(String Recipe) {
    _Recipe = Recipe;
  }
  result_type _Results;

  result_type get Results => _Results;

  set Results(result_type Results) {
    _Results = Results;
  }
  Image _Img;

  Image get Img => _Img;

  set Img(Image Img) {
    _Img = Img;
  }

  dish(this._ID, this._Name, this._Ingredients, this._Recipe, this._Results,
      this._Img);

}